spk/ps3netsrv/src/service-setup.sh

```diff
@@ -1,13 +1,9 @@
 CFG_FILE="${SYNOPKG_PKGDEST}/var/ps3netsrv.conf"
-BIN="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
+BIN_FILE="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
 
-if [ -r "${CFG_FILE}" ]; then
-    . "${CFG_FILE}"
-fi
+export PID_FILE LOG_FILE CFG_FILE BIN_FILE
```

Comment:
Please move all these variables to your starter script and remove export

Reply:
The starter script doesn't have access to these variables without export.